test/forms/DateMappingsSpec.scala

```diff
@@ -435,4 +435,25 @@ class DateMappingsSpec
       filledForm("value.year").value.value mustEqual date.getYear.toString
     }
   }
+
+  "must bind valid month data with short and full month names" in {
+
+    val validMonths = Table(
```

Comment:
For completeness I would add a scenario where some invalid month textual values fail to bind. E.g.: `Febru`, `Anytext`.
But only do that if you think too that it would be useful.

Reply:
This has been addressed